test/bdd/fixtures/agent-rest/docker-compose.yml

```diff
@@ -252,6 +252,58 @@ services:
     networks:
       - bdd_net
 
+  couchdb.example.com:
+    container_name: couchdb.example.com
+    image: ${COUCHDB_IMAGE}:${COUCHDB_IMAGE_TAG}
+    ports:
+      - ${COUCHDB_PORT}:${COUCHDB_PORT}
+    environment:
+      - COUCHDB_USER=${COUCHDB_USERNAME}
+      - COUCHDB_PASSWORD=${COUCHDB_PASSWORD}
+    volumes:
+      - ./couchdb-config/10-single-node.ini:/opt/couchdb/etc/local.d/config.ini
+    networks:
+      - bdd_net
+
+  kms.example.com:
```

Comment:
new container requires adding the following line in `/etc/hosts` to successfully run bdd tests:
`127.0.0.1 kms.example.com`

Reply:
Please document this in `docs/test/bdd_tests.md`

Reply:
No need you can use localhost

Reply:
I tried using localhost, but the key server creating a user keystore returns a valid URL representing the key server host URL. @aholovko do you think we can tell the key server to return URLs with `localhost` from docker-compose.yaml?

Reply:
@Baha-sk you can try to use "KMS_BASE_URL" env variable (https://github.com/trustbloc/hub-kms/blob/master/cmd/kms-rest/startcmd/start.go#L38) and set localhost there or just omit that var and get "naked" url back (without host and port) and prepend it with what you have in the test case.

Reply:
I've updated `KMS_BASE_URL` with localhost and the tests seems to pass, there's no need to add a new entry in `/etc/hosts` @fqutishat @aholovko @llorllale